manuscript.Rmd

```diff
@@ -88,47 +88,47 @@ knitr::opts_chunk$set(cache.extra = knitr::rand_seed)
 
 # Introduction
 
-Based on four principles of a more responsible research assessment in academic hiring and promotion processes [@schonbrodt2022responsible], @gartner2022responsible suggested a specific evaluation scheme for published manuscripts, reusable datasets, and research software.
-We specifically endorse the increased emphasis on research software as a recognizable and laudable scientific contribution.
+Based on four principles of a more responsible research assessment in academic hiring and promotion processes [@schonbrodt2022responsible], @gartner2022responsible suggested a concrete evaluation scheme for published manuscripts, reusable datasets, and research software.
+We strongly support the increased emphasis on research software as a creditable and commendable scientific contribution.
 Why are research software contributions important scientific contributions?
-We would like to respond with a quote from the Science Code Manifesto (http://sciencecodemanifesto.org/):
+We would like to respond with a quote from the Science Code Manifesto [@manifesto]:
 "Software is a cornerstone of science. Without software, twenty-first century science would be impossible."
 and
 "Software is an essential research product, and the effort to produce, maintain, adapt, and curate code must be recognized."
 However, despite the heavy reliance on computational infrastructure, the current academic infrastructure does not adequately incentivise software development and, specifically, good software engineering practice [@baxter2012research].
-In line with principle 3 of @schonbrodt2022responsible, we suggest that criteria for research software contributions must capture two major dimensions: rigor and impact.
+In line with principle 3 of @schonbrodt2022responsible, we suggest that criteria for research software contributions must capture two major dimensions: rigour and impact.
```

Comment:
I generally don't care about american vs. british english (FYI: to be in accordance with APA, you would have to use american english), but I think it's a bit funny to make rigor -> rigour since this is a concept introduced by the paper we answer to, and they use it with the spelling "rigor".

Reply:
ok, so let's switch to American English then for consistency.